sdmetrics/multi_table/multi_single_table.py

```diff
@@ -67,12 +68,19 @@ def _compute(self, real_data, synthetic_data, metadata=None):
             metadata = metadata.to_dict()
 
         values = []
+        errors = {}
         for table_name, real_table in real_data.items():
             synthetic_table = synthetic_data[table_name]
             table_meta = metadata['tables'][table_name]
 
-            score = self.single_table_metric.compute(real_table, synthetic_table, table_meta)
-            values.append(score)
+            try:
+                score = self.single_table_metric.compute(real_table, synthetic_table, table_meta)
+                values.append(score)
+            except IncomputableMetricError as e:
```

Comment:
This should probably not be a single char variable. We can call it `error` or `ime`.